vendor/assets/stylesheets/ombulabs/styleguide/4-molecules/_tables.scss

```diff
@@ -0,0 +1,60 @@
+table{
+        width: 100%;
```

Comment:
There are some things going on with the spacing in this section. Could you fix that? :)